README.md

```diff
@@ -4,11 +4,11 @@ Repository containing monthly challenges in quantum computing.
 
 ## Current Challenge
 
-**Link to Problem Notebook**: [Challenge 02](https://nbviewer.jupyter.org/github/qosf/monthly-challenges/blob/main/challenge-02/qosf-monthly-challenge-02.ipynb)
+**Link to Problem Notebook**: [Challenge 03](https://github.com/qosf/monthly-challenges/blob/main/challenge-02/qosf-monthly-challenge-02.ipynb)
```

Comment:
Needs to be updated to point to challenge-03

Reply:
Updated. Should I add an NBViewer link instead? I wasn't sure how to do it without having the changes merged in first.